featurebyte/service/external_feature_store.py

```diff
@@ -0,0 +1,335 @@
+"""
+External Feature Store Service
+"""
+from typing import List, Optional, cast
+
+import os
+from dataclasses import dataclass
+
+import pandas as pd
+from databricks import sql as databricks_sql
+from databricks_cli.clusters.api import ClusterApi
+from databricks_cli.jobs.api import JobsApi
+from databricks_cli.runs.api import RunsApi
+from databricks_cli.sdk.api_client import ApiClient
+
+from featurebyte.exception import (
+    InvalidFeatureStoreArgumentError,
+    InvalidFeatureStoreClusterNameError,
+    InvalidFeatureStoreJobError,
+)
+from featurebyte.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass
```

Comment:
is this `dataclass` annotation necessary?

also, how is this service going to be used? is it just an internal script, or something that will be called by a SDK interface?